src/Apps/CollectorProfile/collectorProfileRoutes.tsx

```diff
@@ -131,6 +141,13 @@ export const collectorProfileRoutes: AppRouteConfig[] = [
       }
     `,
     children: [
+      {
+        path: "artists",
```

Comment:
The route should be: `/collector-profile/artists`, not `/collector-profile/my-collection/artists`, to match the other routes.